P5/Source/Guidelines/en/DI-PrintDictionaries.xml

```diff
@@ -2653,30 +2653,33 @@ sometimes informally stated values for the particular descriptive features, the
 of the content and of the container objects becomes relevant, in view of the growing
 tendency to interlink pieces of information across Internet resources. In such
 cases, it becomes crucial to be able to encode the fact that whether the information on,
-for instance, the value of the grammatical category of Number is provided as "sg.", "sing.", "Singular", or
-equivalently "poj." in Polish, or "Ez." in German, etc., what is actually referred to is always the same 
+for instance, the value of the grammatical category of Number is provided as <q>sg.</q>, <q>sing.</q>, <q>Singular</q>, or
+equivalently <q>poj.</q> in Polish, or <q>Ez.</q> in German, etc., what is actually referred to is always the same 
 grammatical value that can be rendered with a plethora of markers, depending on the publisher, language, or lexicographic tradition.
 In order to signal that this variety of surface markers in fact indicate the same
 underlying value, it is possible to align them with an external inventory of standardized
-values. The TEI provides means to align grammatical categories as well as their content
-with the ISOcat reference, which is a Web implementation of <ref target="#ISO-12620">ISO 12620</ref>.</p>
+values. The TEI provides the <ident type="class">att.datcat</ident> attribute class for the purpose of aligning grammatical (or indeed any sort of) categories as well as their values
+with a reference taxonomy of shared data categories.</p>
 <p>In the example below, a fragment of the entry for <foreign>isotope</foreign> cited
-in section <ptr target="#DITPGR"/> is adorned by references to ISOcat definitions for "part
-of speech" (<att>dcr:datcat</att>) and "adjective" (<att>dcr:valueDatcat</att>). Depending
+in section <ptr target="#DITPGR"/> is adorned by references to standardised definitions for <soCalled>part
+of speech</soCalled> (<att>datcat</att>) and <soCalled>adjective</soCalled> (<att>valueDatcat</att>). Depending
```

Comment:
Syd, you're being so delightfully Socratic here... ;-) This item relates to my doubts and musings in #2358 , where for a while I considered using `soCalled` together with `term`, but then realised that my reasons have to do with the rendering of these elements rather than their semantic import.

Reply:
replaced with `term`